src/utils/staticTokenDefinition.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  Address,
+  BigInt,
+} from "@graphprotocol/graph-ts"
+  
+// Initialize a Token Definition with the attributes
+export class StaticTokenDefinition {
+  address : Address
+  symbol: string
+  name: string
+  decimals: BigInt
+
+  // Initialize a Token Definition with its attributes
+  constructor(address: Address, symbol: string, name: string, decimals: BigInt) {
+    this.address = address
+    this.symbol = symbol
+    this.name = name
+    this.decimals = decimals
+  }
+
+  // Get all tokens with a static defintion
+  static getStaticDefinitions(): Array<StaticTokenDefinition> {
+    let staticDefinitions = new Array<StaticTokenDefinition>(6)
+
+    // Add DGD
```

Comment:
I understand you got this from Uniswap, and they implemented it for some reason. It would be nice to comment what was this reason, and point to their code as a reference